webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -31,8 +31,12 @@ export const getNotifications = url => dispatch => {
           });
           resolve();
         },
-        () => reject()
-      );
+        (error) => {
```

Comment:
i think we should structure it a bit nicer - make an error function, a success function, and a finally function to make it cleaner.